paymentsheet/src/androidTest/java/com/stripe/android/paymentsheet/PaymentSheetPage.kt

```diff
@@ -60,6 +87,12 @@ internal class PaymentSheetPage(
     }
 
     fun clickPrimaryButton() {
+        composeTestRule.waitUntil(5_000) {
+            composeTestRule
+                .onAllNodes(hasTestTag(PAYMENT_SHEET_PRIMARY_BUTTON_TEST_TAG).and(isEnabled()))
+                .fetchSemanticsNodes().isNotEmpty()
+        }
```

Comment:
Needed to add a delay to account for the email debouncer in Link signup. The ideal solution would be to have `UnconfinedTestDispatcher` handle this since it immediately skips delays since we are testing the full `PaymentSheet` flow, we have no way of injecting this dispatcher into the view models. Using `CoroutineTestRule` fails the test because the view hierarchy is created by the original `Dispatchers.Main` irregardless of the dispatcher set by `Dispatchers.setMain`.

```
android.view.ViewRootImpl$CalledFromWrongThreadException: Only the original thread that created a view hierarchy can touch its views. Expected: main Calling: DefaultDispatcher-worker-2
```